[2020-03-27 15:46:42] [[31merror[39m] Error: ER_BAD_DB_ERROR: Unknown database 'qldv1'
    at Handshake.Sequence._packetToError (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Handshake.ErrorPacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/sequences/Handshake.js:123:18)
    at Protocol._parsePacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    --------------------
    at Protocol._enqueue (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at Connection.connect (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:116:18)
    at Object.<anonymous> (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/database/connect_db.js:9:9)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
[2020-03-27 15:47:13] [[31merror[39m] Error: ER_BAD_DB_ERROR: Unknown database 'qldv1'
    at Handshake.Sequence._packetToError (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Handshake.ErrorPacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/sequences/Handshake.js:123:18)
    at Protocol._parsePacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    --------------------
    at Protocol._enqueue (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at Connection.connect (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:116:18)
    at Object.<anonymous> (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/database/connect_db.js:9:9)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
[2020-03-27 15:47:19] [[31merror[39m] Error: Cannot enqueue Query after fatal error.
    at Protocol._validateEnqueue (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:212:16)
    at Protocol._enqueue (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:138:13)
    at Connection.query (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:198:25)
    at Promise (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/controller/controller.js:7:16)
    at new Promise (<anonymous>)
    at index (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/controller/controller.js:6:13)
    at Layer.handle [as handle_request] (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/express/lib/router/layer.js:95:5)
    at next (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/express/lib/router/route.js:137:13)
    at middleware (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/middleware/middleware.js:11:5)
    at Layer.handle [as handle_request] (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/express/lib/router/layer.js:95:5)
[2020-03-27 15:49:14] [[31merror[39m] Error: ER_BAD_DB_ERROR: Unknown database 'qldv1'
    at Handshake.Sequence._packetToError (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Handshake.ErrorPacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/sequences/Handshake.js:123:18)
    at Protocol._parsePacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    --------------------
    at Protocol._enqueue (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at Connection.connect (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:116:18)
    at Object.<anonymous> (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/database/connect_db.js:9:9)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
[2020-03-27 15:52:05] [[31merror[39m] Error: ER_NO_SUCH_TABLE: Table 'qldv.test' doesn't exist
    at Query.Sequence._packetToError (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    --------------------
    at Protocol._enqueue (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Connection.query (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:198:25)
    at Promise (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/controller/controller.js:7:16)
    at new Promise (<anonymous>)
    at index (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/controller/controller.js:6:13)
    at Layer.handle [as handle_request] (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/express/lib/router/layer.js:95:5)
    at next (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/express/lib/router/route.js:137:13)
    at middleware (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/middleware/middleware.js:11:5)
    at Layer.handle [as handle_request] (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/express/lib/router/layer.js:95:5)
    at next (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/express/lib/router/route.js:137:13)
[2020-03-27 15:58:44] [[31merror[39m] Error: ER_NO_SUCH_TABLE: Table 'qldv.test' doesn't exist
    at Query.Sequence._packetToError (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    --------------------
    at Protocol._enqueue (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Connection.query (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/mysql/lib/Connection.js:198:25)
    at Promise (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/controller/controller.js:7:16)
    at new Promise (<anonymous>)
    at index (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/controller/controller.js:6:13)
    at Layer.handle [as handle_request] (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/express/lib/router/layer.js:95:5)
    at next (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/express/lib/router/route.js:137:13)
    at middleware (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/middleware/middleware.js:11:5)
    at Layer.handle [as handle_request] (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/express/lib/router/layer.js:95:5)
    at next (/media/quangmv/DATA/ACEXIS/JavaScript30/node_demo/node_modules/express/lib/router/route.js:137:13)
